src/ralph/backends/lrs/fs.py

```diff
@@ -53,7 +53,9 @@ def write(  # noqa: PLR0913
         target = target if target else self.settings.DEFAULT_LRS_FILE
```

Comment:
```suggestion
        if target:
            target = str(Path(target) / Path(self.settings.DEFAULT_LRS_FILE))
        else:
            target = self.settings.DEFAULT_LRS_FILE
```
If I understand it right, with the `target` argument, we now read the `default_lrs_file` in a target directory:
```bash
- fs_lrs.jsonl  # without target
- target_A
    - fs_lrs.jsonl
- target_B
    - fs_lrs.jsonl
```
However, when we write - the target is the (destination) file, right?

Reply:
That's true, good point! Fixed it with your suggestion